background_r: 255
background_g: 255
background_b: 0
